cvat-ui/src/components/dashboard/content/dashboard-content.tsx

```diff
@@ -79,15 +81,13 @@ class DashboardContent extends Component<any, any> {
   }
 
   private onDeleteTask(task: any) {
-    const props = this.props;
-
     confirm({
       title: 'Do you want to delete this task?',
       okText: 'Yes',
       okType: 'danger',
       centered: true,
       onOk() {
-        return props.deleteTask(task);
+        return task.delete();
```

Comment:
It is async function. What if promise is rejected? Will it be handled?

Reply:
This is not a final solution. I'm working on this part right now.

Reply:
Could you please add TODO marker into the code?